README.md

```diff
@@ -22,6 +22,12 @@ cd starshot && ddev install
 ```
 You'll need DDEV 1.23.0 or later. [See the documentation](https://ddev.readthedocs.io/en/stable/users/install/ddev-upgrade/) if you need to upgrade.
 
+If you use [Lando](https://lando.dev/), you can get Starshot up and running with this:
+```
+git clone https://github.com/phenaproxima/starshot-prototype.git starshot
+cd starshot && lando start && lando install && lando login
```

Comment:
This is, essentially, _four_ commands separated by &&, which is a bit...twitchy. Is there any way to script this as truly a single command, as I did with `.ddev/commands/host/install`?

Also, if `lando install` is already a thing, maybe we should rename both commands to `install-starshot` for consistency.

Reply:
Agree `lando install` is not clever.

I can reduce this to 2 commands : `lando start` and `lando drupal:install`

I've updated the README accordingly